product_carousel_image_attachment/models/product_image.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2024 Quartile
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class ProductImage(models.Model):
+    _inherit = "product.image"
+
+    @api.model
+    def _cron_resize_product_image(self, limit):
+        images = self.sudo().search([], limit=limit)
```

Comment:
@AungKoKoLin1997 Would it be possible to filter records by size? There can be so many records unnecessarily processed otherwise. We could live with this if it's hard to do so.